antora.yml

```diff
@@ -12,22 +12,22 @@ asciidoc:
     serverlessoperatorname: OpenShift Serverless Operator
     serverlessproductname: OpenShift Serverless
     product_name: OpenShift Serverless Logic
-    kogito_version_redhat: 1.38.0.Final-redhat-00002
+    kogito_version_redhat: 1.40.0.Final-redhat-00009
     operator_name: Kogito Serverless Workflow Operator
     quarkus_platform: com.redhat.quarkus.platform
     kogito_sw_ga: >-
       org.kie.kogito:kogito-quarkus-serverless-workflow
-    quarkus_version: 2.13.7.Final-redhat-00003
-    quarkus_platform_version: 2.13.7.Final-redhat-00003
+    quarkus_version: 2.13.8.Final-redhat-00005
+    quarkus_platform_version: 2.13.8.SP2-redhat-00001
     java_min_version: 11+
     maven_min_version: 3.8.6
     graalvm_min_version: 22.3.0
     spec_version: 0.8
     vscode_version: 1.66.0
-    kn_cli_version: 0.28.0
+    kn_cli_version: 0.30.3
```

Comment:
@domhanak This is version from the TP1. I guess the SP1 has the same one.

Reply:
Yes, the  correct version  of quarkus used in kn cli is injected during the openshift-knative build process